packages/developer-portal/src/components/apps/pipeline/pipeline-deployments-table.tsx

```diff
@@ -141,7 +141,13 @@ export const PipelineDeploymentTable: FC = () => {
                   },
                   {
                     label: 'Status',
-                    value: buildStatusToReadable(buildStatus as string),
+                    value: '',
+                    children: (
+                      <>
+                        <StatusIndicator intent={buildStatusToIntent(buildStatus as string)} />
+                        {buildStatusToReadable(buildStatus as string)}
+                      </>
```

Comment:
Perhaps we should make this a reusable component as we've used it in many places

Reply:
Maybe? Not sure, only 2 places I think

Reply:
quiet a few I've seen, doesn't matter we can do it another time :)